kubebench/base/config-map.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubebench-config
+data:
+  kubebenchconfig.yaml: |
+    """
+    defaultWorkflowAgent:
+      container:
+        name: kubebench-workflow-agent
+        image: ciscoai/kubebench-workflow-agent:latest

```

Comment:
is this image official released? can we make it configurable by env or else?

Reply:
I updated it to officially released